wgsl/index.bs

```diff
@@ -12689,6 +12689,13 @@ WGSL does not specify a [=ieee754/rounding mode=].
 Note: Floating point types include positive and negative infinity, so
 the correctly rounded result may be finite or infinite.
 
+Note: The result of an operation when computed with unbounded precision may require precision
+which exceeds that of a double. An example of such a case is in `x - y` where `x=1.0`
+and `y=1.17e-38` (smallest positive normal single precision float). These numbers have
+exponents are that are `126` units apart. Double precision only has `52` explicit bits of mantissa,
+so in doing the subtraction, all the significant bits of `y` are lost. Dependening on rounding modes
+(JS uses round-to-even) we can get 'x - y = x' for these (and many other) numbers.
```

Comment:
Write `[[ECMASCRIPT]]` instead of `JS` here. The spec already references ECMAScript in a non-normative way (as it is referenced here).
Also, I'd move this parenthetical remark to its own sentence afterward, to imrpove the flow:

> Depending on rounding modes, for this and many other cases where `y` is small but non-zero, the WGSL expression `x - y` may yield the same value as `x`.
> Note that [[ECMAScript]] uses the equivalent of the [[!IEEE-754|IEEE-754]] roundTiesToEven rounding mode.